plugins/nisq_analyzer.py

```diff
@@ -114,7 +125,7 @@ def get(self):
 class ProcessView(MethodView):
     """Start a long running processing task."""
 
-    @NISQ_BLP.arguments(NisqAnalyzerParametersSchema(unknown=EXCLUDE), location="form")
+    @NISQ_BLP.arguments(NisqAnalyzerParametersSchema(unknown=EXCLUDE), location="json")
```

Comment:
If you use `unknown=INCLUDE` instead, then you can use an empty schema and do not need to wrap the incoming json in an envelope. Of course, a schema that actually validates the nisq analyzer results would be even better.
(see <https://marshmallow.readthedocs.io/en/stable/quickstart.html#handling-unknown-fields>)